### PR TITLE
Add affiliation for ARX

### PR DESCRIPTION
**Pull request by jnear**
**One sentence description of contribution:**
Add affiliation for the ARX tool.

**Additional notes:**
Needs approval by @prasser

### CODE REVIEW

File: tools/de-identification/ARX/README.md [hunk 1]
```diff
@@ -8,4 +8,6 @@
 
 **GitHub User Serving as POC:** @prasser
 
+**Affiliation/Organization(s) Contributing:** TUM – Technische Universität München
```
Comment (prasser) [inline tools/de-identification/ARX/README.md]: Thanks. Please replace by "Technical University of Munich".

On a side note: are there any guidelines on what is considered a "contributing organization"? This is an open source project and there are quite a few organizations and also companies that have contributed to the project in the past...

Comment (jnear) [reply tools/de-identification/ARX/README.md]: Fixed! Let me look into the contributing organization issue.

Comment (jnear) [reply tools/de-identification/ARX/README.md]: @prasser It's up to you which organizations to include here. Would you like me to add some more? Or add a link to a page of yours that lists contributors?

Comment (prasser) [reply tools/de-identification/ARX/README.md]: @jnear It's OK for now. I'll think about it and maybe create a new PR if I find a good solution.

**Pull request merged**